src/web/app/pages-instructor/instructor-sessions-page/instructor-sessions-page.component.html

```diff
@@ -2,9 +2,20 @@
   You have not created any courses yet, or you have no active courses. Go <a href="#" (click)="createOrUnarchiveCourse(); $event.preventDefault()">here</a> to create or unarchive a course.
 </div>
 
-<tm-session-edit-form [formMode]="SessionEditFormMode.ADD" [(model)]="sessionEditFormModel"
-                      [courseCandidates]="courseCandidates" [templateSessions]="templateSessions"
-                      (addNewSessionEvent)="addNewSessionHandler()" (copyOtherSessionsEvent)="copyFromOtherSessionsHandler()"></tm-session-edit-form>
+<div>
+  <button class="btn btn-primary" (click)="isSessionEditFormExpanded=true"
```

Comment:
```suggestion
  <button class="btn btn-primary" (click)="isSessionEditFormExpanded = true"
```

Mind the whitespaces.